### ISSUE TITLE
Submit editing messages on enter 

### ISSUE THREAD

**Issue by mxstbr** [withspectrum/spectrum#4747]
I think consistency is a fair point here, we should always send messages on enter and add new lines on shift+enter. Right now, that behaviour is inverted for editing messages, which is strange: https://spectrum.chat/spectrum/general/return-button-behavior~cd72b969-28dc-4334-945d-2ea5c4859cfc

**Comment by brianlovin** [comment]
It's inverted because the edit message box looks like a textarea for a comment and the chat input looks like...a chat input. The design of the input itself has some affordance for how it should behave.

We can make them the same, but honestly I'd probably prefer to move the chat input to behave more like a textarea/comment. Only because right now it's incredibly tedious to type code blocks in the chat input...

**Comment by mirko77** [comment]
Inverting behavior is counter-intuitive. 
Moreover, any other app I checked is shift+enter new line, enter to send/save

**Comment by brianlovin** [comment]
> Inverting behavior is counter-intuitive.

Agreed, we just need to align the actual design of the message composer/editor :)

**Comment by lookapanda** [comment]
As a side note: Discord (I know, I use that as a reference a lot :P) is doing it like this:

Enter to send, Shift+Enter to add a new line but it detects if you're inside a code block (e.g. between a triple back tick) and if that's the case, pressing enter will add a new line instead.

**Comment by Tbhesswebber** [comment]
I tried to find other conversations surrounding any supersets of this topic and couldn't find any... has there been any traction one way or the other on this?

I only ask because, if the behavior or a setting to modify the behavior may be coming, I will patiently wait.  If not, I plan to write my own scripts that override the behaviors of the chat input on desktop.
